src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
There is no standard unwind information on Windows x86. In CoreCLR, we just store the unwind information as part of the GCInfo on x86.